motor_joints:
  robot_movement_joints:
    left_front_joint: &movement_left_front_joint p3at_front_left_wheel_joint
    right_front_joint: &movement_right_front_joint p3at_front_right_wheel_joint
    left_rear_joint: &movement_left_rear_joint p3at_back_left_wheel_joint
    right_rear_joint: &movement_right_rear_joint p3at_back_right_wheel_joint

joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

movement_left_front_joint_velocity_controller:
  type: velocity_controllers/JointVelocityController
  joint: *movement_left_front_joint

movement_right_front_joint_velocity_controller:
  type: velocity_controllers/JointVelocityController
  joint: *movement_right_front_joint
  
movement_left_rear_joint_velocity_controller:
  type: velocity_controllers/JointVelocityController
  joint: *movement_left_rear_joint
  
movement_right_rear_joint_velocity_controller:
  type: velocity_controllers/JointVelocityController
  joint: *movement_right_rear_joint
